core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -485,12 +731,31 @@ object ConsumerGroupCommand extends Logging {
     case object Ignore extends LogEndOffsetResult
   }
 
+  sealed trait LogStartOffsetResult
```

Comment:
Seems kind of unnecessary to three classes which do the same thing, just named differently. Maybe we can just have a single `LogOffsetResult`? I think that will remove a lot of boilerplate conversion above.